test/integ-new/main_new_test.go

```diff
@@ -180,6 +182,53 @@ func TestAPIAndPublicKeyService(t *testing.T) {
 	}
 }
 
+func TestUIService(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name           string
+		wantStatusCode int
+	}{
+		{
+			name:           "test_health_checkpoint",
+			wantStatusCode: 200,
+		},
+	}
+
+	for _, tc := range cases {
```

Comment:
We don't need this table structure since we only have one test case.

Reply:
I think we can also put the cert rotator case in here since it also checks the return is 200

Reply:
Update: removed the for loop.